partner_multi_relation_extended/models/res_partner_relation_all.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+# © 2017 Savoir-faire Linux
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import AccessError
+
+
+class ResPartnerRelationAll(models.AbstractModel):
+
+    _inherit = 'res.partner.relation.all'
+
+    strength = fields.Many2one(
+        string='Strength',
+        comodel_name='res.partner.relation.strength',
+    )
+
+    note = fields.Char(
+        'Note',
+        help='Use this field to add information about the relation. For '
+             'example, if the relation is "Belongs to the professional order",'
+             ' you can put here the professional order number.',
+    )
+
+    is_automatic = fields.Char(
+        'Automatic',
+        readonly=True,
+        help='This relation has been automatically created by the system. '
+             'Only the system administrator can update or delete it.',
+    )
+
+    @api.model_cr_context
+    def _auto_init(self):
+        """
+        Add new fields to auto_init
+        """
+        self._additional_view_fields.append('strength')
+        self._additional_view_fields.append('note')
+        self._additional_view_fields.append('is_automatic')
```

Comment:
@Ehtaga what is this for ?

Reply:
In the original partner_multi_relation_module:
``` 
_additional_view_fields = []
"""append to this list if you added fields to res_partner_relation that
you need in this model and related fields are not adequate (ie for sorting)
You must use the same name as in res_partner_relation.
Don't overwrite this list in your declaration but append in _auto_init:

@api.model_cr_context
def _auto_init(self):
    self._additional_view_fields.append('my_field')
    return super(ResPartnerRelationAll, self)._auto_init()

my_field = fields...
"""```

Reply:
@Ehtaga ok thanks

Reply:
@Ehtaga ok thanks